### DISCUSSION TITLE
Baseline Software Versions

### DISCUSSION THREAD

**Discussion by hughgs** [durhambikecoop/freehub#1] [General]
Looking at the different sites, what do you think about

Rails  -  7.0.4.1
Ruby  -  3.2
MySQL  -  8.0.32

I didn't see an official 7.1 for Rails (https://rubyonrails.org/category/releases)

**Comment by hughgs** [comment]
We should also stack hands on the latest gems and their versions.

**Comment by greasysock** [comment]
At the moment I am working with:
  * Ruby 3.2.0
  * Rails 7.0.4
  * PostgreSQL 14.6 for the development database
  * MySQL `mysql  Ver 15.1 Distrib 10.9.4-MariaDB`, MariaDB, for migrating from the old database dump to the new database

**Reply by hughgs** [reply]
Sounds good. I think that if I just keep track of your Gemfile I should be good. Especially since I'm avoiding too much in the way of code changes. If I do push something to main just overwrite my Gemfile, since your Gemfile should be the requirement.
